src/add-remove.js

```diff
@@ -0,0 +1,100 @@
+  list, save, fixIndex, setList, remove,
+} from './status-update';
+
+const todoList = document.getElementById('todo-list');
+export const items = document.getElementsByClassName('task-item');
+
+const createTask = (task) => {
+  let todoObj = '';
+  if (task.completed === true) {
+    todoObj = `
+      <article id="${task.index}" class="task-item">
+        <input type='checkbox' name='completed' class="checkbox" checked>
+        <span class='task-description completed' id="desc-${task.index}" contenteditable>${task.description}</span>
+        <i class="bi bi-three-dots-vertical"></i>
+        <i class="bi bi-trash"></i>
+      </article>`;
+  } else {
+    todoObj = `
+        <article  id="${task.index}" class="task-item">
+          <input type='checkbox' name='completed' class="checkbox">
+          <span class='task-description' id="desc-${task.index}" contenteditable>${task.description}</span>
+          <i class="bi bi-three-dots-vertical"></i>
+          <i class="bi bi-trash"></i>
+        </article>`;
+  }
+
+  todoList.innerHTML += todoObj;
+  checkboxesEvent();
+};
+
+export const displayTasks = () => {
+  todoList.innerHTML = '';
+
+  const sortedList = list.sort((a, b) => {
+    if (a.index > b.index) {
+      return 1;
+    }
+    if (a.index < b.index) {
+      return -1;
+    }
+    return 0;
+  });
+
+  sortedList.forEach((task) => {
+    createTask(task);
+  });
+  checkboxesEvent();
+  save();
+};
+
+export function edit() {
+  const editables = document.querySelectorAll('[contenteditable]');
+  for (let i = 0; i < editables.length; i += 1) {
+    editables[i].addEventListener('click', () => {
+      editables[i].parentNode.children[3].classList.add('show');
+      editables[i].parentNode.children[2].classList.add('hide');
+      editables[i].addEventListener('blur', (event) => {
+        if (event.target) {
+          localStorage.setItem('edit', JSON.stringify(editables[i].innerHTML));
+          list[i].description = JSON.parse(localStorage.getItem('edit'));
+          save();
+        }
+        // editables[i].parentNode.children[3].classList.remove('show')
+        editables[i].parentNode.children[2].classList.remove('hide');
+        edit();
+        remove();
+      });
+    });
+  }
+}
+
+export function clear() {
+  document.getElementById('clear-complete').addEventListener('click', () => {
+    const callback = (task) => task.completed === false;
+    const todo = list.filter(callback);
+    setList(todo);
+    fixIndex(list);
+    checkboxesEvent();
+    save();
+  });
+}
+
+export function add() {
+  document.getElementById('task-entry').addEventListener('keypress', (event) => {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      const description = document.getElementById('task-entry');
+      const id = list.length;
+      const task = { description: description.value, completed: false, index: id };
+      description.value = '';
+      list.push(task);
+      save();
```

Comment:
- [ ] Please change these and all other regular functions to es6 arrow function syntax.

Reply:
This issue is not fully resolved.